toolkit/sidechain/domain/src/byte_string.rs

```diff
@@ -1,9 +1,10 @@
-use core::fmt::{Debug, Display};
-use core::ops::Deref;
+//! Module providing various binary data types for use by the Partner Chains Toolkit crates
```

Comment:
What a "binary" data type is? Definitions in this file define Rust types and also their serialization.